### PR TITLE
Mark many strings as non-translatable

### PR DESCRIPTION
**Pull request by dbrgn**
When creating a new dialog, please make sure to remove the "translatable" checkbox when necessary.

### CODE REVIEW

**Comment by dbrgn** [comment]
Any idea why the tests fail?

**Comment by ubruhin** [comment]
```
In file included from ../../../../libs/librepcb/libraryeditor/newelementwizard/newelementwizardpage_entermetadata.cpp:24:0:
/home/travis/build/LibrePCB/LibrePCB/build/libs/librepcb/libraryeditor/ui_newelementwizardpage_entermetadata.h: In member function ‘void librepcb::library::editor::Ui_NewElementWizardPage_EnterMetadata::retranslateUi(QWizardPage*)’:
/home/travis/build/LibrePCB/LibrePCB/build/libs/librepcb/libraryeditor/ui_newelementwizardpage_entermetadata.h:156:25: error: ‘class QPlainTextEdit’ has no member named ‘setPlaceholderText’
         edtDescription->setPlaceholderText(QString());
                         ^
```

Probably this was introduced with [this change](https://github.com/LibrePCB/LibrePCB/pull/301/files#diff-f052b3407e7162f97311460e2d480090R77)(?).

**Comment by dbrgn** [comment]
> Are these all strings which shouldn't be translated, or just a "random" selection of them?

All that I found in the dialogs. I went through all dialogs I think (unless I missed some).

> Probably this was introduced with this change(?).

Oh, whoops, what does that change do? Must have been something done by the designer. I'll revert that line.

File: apps/librepcb/firstrunwizard/firstrunwizardpage_workspacepath.ui [hunk 1]
```diff
@@ -14,7 +14,7 @@
    <string>WizardPage</string>
   </property>
   <property name="title">
-   <string>Select Workspace Path</string>
+   <string>Select Wor&amp;kspace Path</string>
```
Comment (dbrgn) [inline apps/librepcb/firstrunwizard/firstrunwizardpage_workspacepath.ui]: I just noticed these changes now. Do we want them?

File: libs/librepcb/libraryeditor/dev/deviceeditorwidget.ui [hunk 2]
```diff
@@ -426,6 +426,7 @@
  </customwidgets>
  <resources>
   <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
```
Comment (dbrgn) [inline libs/librepcb/libraryeditor/dev/deviceeditorwidget.ui]: This one is strange as well.

File: apps/librepcb/firstrunwizard/firstrunwizardpage_workspacepath.ui (hunk 1 above)
Comment (ubruhin) [reply apps/librepcb/firstrunwizard/firstrunwizardpage_workspacepath.ui]: Yes you can keep them, these are the automatically created keyboard shortcuts (ALT+k in this case). Although I wonder why this is created for the title of a wizard page... ;)

File: libs/librepcb/projecteditor/newprojectwizard/newprojectwizardpage_metadata.ui [hunk 3]
```diff
@@ -213,6 +213,7 @@
  </widget>
  <resources>
   <include location="../../../../img/images.qrc"/>
+  <include location="../../../../img/images.qrc"/>
```
Comment (ubruhin) [inline libs/librepcb/projecteditor/newprojectwizard/newprojectwizardpage_metadata.ui]: same here...

**Comment by ubruhin** [comment]
Thanks @dbrgn :+1:

**Pull request merged**